sci-electronics/kicad-symbols/kicad-symbols-9999.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Electronic Schematic and PCB design tools symbol libraries"
+HOMEPAGE="https://gitlab.com/kicad/libraries/kicad-symbols"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://gitlab.com/kicad/libraries/kicad-symbols.git"
+	inherit git-r3
+	# x11-misc-util/macros only required on live ebuilds
+	LIVE_DEPEND=">=x11-misc/util-macros-1.18"
+else
+    MY_PV="${PV/_rc/-rc}"
+	MY_P="${PN}-${MY_PV}"
+	SRC_URI="https://gitlab.com/kicad/libraries/${PN}/-/archive/${MY_PV}/${MY_P}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="CC-BY-SA-4.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=""
+RDEPEND=">=sci-electronics/kicad-5.1.0"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
```

Comment:
Wow, a lot has happened here!

I haven't written live ebuilds and don't know the interactions, but I believe this `S` needs to be defined in the `if-else` block above.
Basically any -9999 ebuilds fail to:
```
>>> Source unpacked in /var/tmp/portage/sci-electronics/kicad-footprints-9999/work
 * ERROR: sci-electronics/kicad-footprints-9999::gentoo failed (prepare phase):
 *   The source directory '/var/tmp/portage/sci-electronics/kicad-footprints-9999/work/kicad-footprints-' doesn't exist
 * 
 * Call stack:
 *            ebuild.sh, line  809:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1059:  Called __dyn_prepare
 *   phase-functions.sh, line  385:  Called die
 * The specific snippet of code:
 *   		die "The source directory '${S}' doesn't exist"
```

```
/var/tmp/portage/vbslogs/kicad-meta-9999-0.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-meta-9999-0.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-meta-9999-0.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-meta-9999-0.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-symbols-9999-3.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-symbols-9999-3.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-templates-9999-2.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-templates-9999-2.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-9999-1.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-footprints-9999-6.json:        "exit_code": 1,
/var/tmp/portage/vbslogs/kicad-footprints-9999-6.json:        "exit_code": 1,
Grepping through the logs for any errors or QA notices...
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132136.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132204.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132152.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132143.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132128.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-templates-9999:20211122-133340.log: * ERROR: sci-electronics/kicad-templates-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132158.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-9999:20211122-132210.log: * ERROR: sci-electronics/kicad-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-footprints-9999:20211122-135501.log: * ERROR: sci-electronics/kicad-footprints-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-symbols-9999:20211122-133350.log: * ERROR: sci-electronics/kicad-symbols-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-symbols-9999:20211122-133408.log: * ERROR: sci-electronics/kicad-symbols-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-templates-9999:20211122-133319.log: * ERROR: sci-electronics/kicad-templates-9999::gentoo failed (prepare phase):
/var/tmp/portage/vbslogs/build/sci-electronics/kicad-footprints-9999:20211122-135511.log: * ERROR: sci-electronics/kicad-footprints-9999::gentoo failed (prepare phase):
```

On a positive note, the _rc ones seem to work!

Reply:
Damn... I had been working with the 9999 for months now and then refactored to get rc1

I'll do the of/else for S as this isn't needed for 9999 as you said